WinUIGallery/Samples/SamplePages/TitleBarWindow.xaml.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Linq;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace WinUIGallery.Samples.SamplePages;
+
+public sealed partial class TitleBarWindow : Window
+{
+    public TitleBarWindow()
+    {
+        this.InitializeComponent();
+        this.ExtendsContentIntoTitleBar = true; // Extend the content into the title bar and hide the default title bar
```

Comment:
```suggestion
        this.ExtendsContentIntoTitleBar = true; // Extend the content into the title bar and hide the default title bar
        this.AppWindow.TitleBar.PreferredHeightOption = Microsoft.UI.Windowing.TitleBarHeightOption.Tall;
```
To be aligned with the height of the title bar
![image](https://github.com/user-attachments/assets/4b951d1b-e188-4fae-8ba2-aa5cb7c83e1f)

Reply:
we need to add this in the sample code also
![image](https://github.com/user-attachments/assets/17178261-b254-4742-bd30-8c59c79f3c8f)

Reply:
Or in XAML (same result)
```xaml
<TitleBar
            Height="{StaticResource TitleBarExpandedHeight}"
```